### PR TITLE
Button styling

### PR DESCRIPTION
**Pull request by epachols**
also in this branch -> refactoring function of join and create feature pages to use <Redirect /> from react-router-dom

### CODE REVIEW

File: src/webapp/features/rabble/Buttons.tsx [hunk 1]
```diff
@@ -58,37 +60,94 @@ const Buttons = ({
...
+        //       "Are you sure you want to Pass your turn?"
+        //     );
+        //     if (confirmed) {
+        //       endTurn();
+        //     }
+        //   }}
+        // >
+        //   <FlagIcon />
+        //   PASS
+        // </button>
```
Comment (apachols) [inline src/webapp/features/rabble/Buttons.tsx]: Ok, in the commit that cleans up your work-zone comments, let's also get rid of these previous button onclick implementations that are commented out.

Nice refactor!

File: src/webapp/features/rabble/components/Theme.tsx [hunk 2]
```diff
@@ -2,7 +2,9 @@ import React, { useState, useEffect } from "react";
 
 const themes = {
   dark: {
-    bg: "#3d3d29",
+    // bg: "#3d3d29",
+    // bg: "#23201c",
+    bg: "#2e353d",
```
Comment (apachols) [inline src/webapp/features/rabble/components/Theme.tsx:7]: Also maybe these commented lines can go?  Unless we need em, up to you

Comment (epachols) [reply src/webapp/features/rabble/components/Theme.tsx:7]: just options open! easily removable, I'll do a deep dive on the theme later, it requires going into each module and using the vars where appropriate.

File: src/webapp/features/rabble/Buttons.tsx (hunk 1 above)
Comment (epachols) [reply src/webapp/features/rabble/Buttons.tsx]: done! I just wanted to give you a chance to review them side-by-side if you thought I needed changes.

**Review by apachols — APPROVED** [review]
LGTM! 👍 Nice work 🙂

**Pull request merged**